code/game/machinery/big_manipulator.dm

```diff
@@ -146,11 +162,77 @@
 		return ITEM_INTERACT_SUCCESS
 	return ITEM_INTERACT_BLOCKING
 
+/obj/machinery/big_manipulator/multitool_act(mob/living/user, obj/item/multitool/tool)
+	if(!panel_open)
+		return ITEM_INTERACT_BLOCKING
+	wires.interact(user)
+	return ITEM_INTERACT_SUCCESS
+
+/obj/machinery/big_manipulator/multitool_act_secondary(mob/living/user, obj/item/tool)
+	return multitool_act(user, tool)
+
+/obj/machinery/big_manipulator/wirecutter_act(mob/living/user, obj/item/tool)
+	if(!panel_open)
+		return ITEM_INTERACT_BLOCKING
+	wires.interact(user)
+	return ITEM_INTERACT_SUCCESS
+
+/obj/machinery/big_manipulator/wirecutter_act_secondary(mob/living/user, obj/item/tool)
+	return wirecutter_act(user, tool)
+
 /obj/machinery/big_manipulator/RefreshParts()
 	. = ..()
 
 	manipulator_lvl()
 
+/obj/machinery/big_manipulator/mouse_drop_dragged(atom/monkey, mob/user, src_location, over_location, params)
+	if(isnull(monkey_worker))
+		return
+	if(on_work)
+		balloon_alert(user, "turn it off first!")
+		return
+	var/mob/living/carbon/human/species/monkey/poor_monkey = monkey_worker.resolve()
```

Comment:
```suggestion
	var/mob/living/carbon/human/species/monkey/poor_monkey = monkey_worker.resolve()
	if(isnull(poor_monkey))
		return
```

This is what weak refs are for. It's `resolve()` proc will return null if the monkey somehow got deleted so check for that